plugins/scaffolder-backend/src/lib/catalog/CatalogEntityClient.ts

```diff
@@ -46,6 +49,11 @@ export class CatalogEntityClient {
     const baseUrl = await this.discovery.getBaseUrl('catalog');
     const response = await fetch(
```

Comment:
We actually want to be able to use the catalog client here, rather than a plain fetch. Same goes for the `auth-backend`

Reply:
Changed CatalogEntityClient and CatalogIdentityClient to wrap CatalogClient